.gitignore

```diff
@@ -1,4 +1,3 @@
-*.py[co]
```

Comment:
I still think we are not going to add these files to git, so should we keep the gitignore?